api/server/backup.go

```diff
@@ -0,0 +1,250 @@
+package server
+
+import (
+	"encoding/json"
+	"github.com/libopenstorage/openstorage/api"
+	"net/http"
+)
+
+func (vd *volAPI) backup(w http.ResponseWriter, r *http.Request) {
```

Comment:
seems like there is a slight inconsistency in the end points w.r.t to params, sometimes they only accept objects, sometimes multiple params are passed separately using key=val form. should we just pick and chose one of these ? (objects are better because adding fields is easier w.r.t to signature change).

Reply:
I see. I 'll object to all the apis.